src/Hostname.php

```diff
@@ -67,6 +67,7 @@ class Hostname extends AbstractValidator
 
     /**
      * Array of valid top-level-domains
+     * Version 2015072300
```

Comment:
We could use here IanaVersion so the word becomes unique and also express this is not our release version